README.md

```diff
@@ -678,6 +700,286 @@ in the Migration dashboard.
+you have code like this:
+
+```python
+for table in spark.catalog.listTables():
+    do_stuff_with_table(table)
+```
+
+you need to make sure that `do_stuff_with_table` can handle the new format.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `dbfs-read-from-sql-query`
+
+DBFS access is not allowed in Unity Catalog, so if you have code like this:
+
+```python
+df = spark.sql("SELECT * FROM parquet.`/mnt/foo/path/to/file`")
+```
+
+you need to change it to use UC tables.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `dbfs-usage`
+
+DBFS does not work in Unity Catalog, so if you have code like this:
+
+```python
+display(spark.read.csv('/mnt/things/e/f/g'))
+```
+
+You need to change it to use UC tables or UC volumes.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `dependency-not-found`
+
+This message indicates that the linter has found a dependency, like Python source file or a notebook, that is not
+available in the workspace. The user must ensure that the dependency is available in the workspace. This usually
+means an error in the user code.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `direct-filesystem-access`
+
+It's not allowed to access the filesystem directly in Unity Catalog, so if you have code like this:
+
+```python
+spark.read.csv("s3://bucket/path")
+```
+
+you need to change it to use UC tables or UC volumes.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `implicit-dbfs-usage`
+
+The use of DBFS is not allowed in Unity Catalog, so if you have code like this:
+
+```python
+display(spark.read.csv('/mnt/things/e/f/g'))
+```
+
+you need to change it to use UC tables or UC volumes.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `jvm-access-in-shared-clusters`
+
+You cannot access Spark Driver JVM on Unity Catalog clusters in Shared Access mode. If you have code like this:
+
+```python
+spark._jspark._jvm.com.my.custom.Name()
+```
+
+or like this:
+
+```python
+log4jLogger = sc._jvm.org.apache.log4j
+LOGGER = log4jLogger.LogManager.getLogger(__name__)
+```
+
+you need to change it to use Python equivalents.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `legacy-context-in-shared-clusters`
+
+SparkContext (`sc`) is not supported on Unity Catalog clusters in Shared access mode. Rewrite it using SparkSession
+(`spark`). Example code that triggers this message:
+
+```python
+df = spark.createDataFrame(sc.emptyRDD(), schema)
+```
+
+or this:
+
+```python
+sc.parallelize([1, 2, 3])
+```
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `not-supported`
+
+Installing eggs is no longer supported on Databricks 14.0 or higher.
+
+[[back to top](#databricks-labs-ucx)]
+
+#### `notebook-run-cannot-compute-value`
+
+Path for `dbutils.notebook.run` cannot be computed and requires adjusting the notebook path.
+It is not clear for automated code analysis where the notebook is located, so you need to simplify the code like:
```

Comment:
"automated code analysis" is unclear to me. Automation is inherently part the code analysis? 🤔 I think it would be more consistent to call this "linting" or "linter" instead.